intra-factory_transport/intra-factory_transport.ipynb

```diff
@@ -10,7 +10,7 @@
     "import os\n",
```

Comment:
Lets add details "!pip install requirements.txt" as one of the cells so notebooks can be installed.

Reply:
May be we should add --quiet to this option but highlight any failures.